setup.py

```diff
@@ -32,7 +32,7 @@ def find_version(*parts):
       long_description=read("README.rst"),
       license="BSD-3-Clause",
       install_requires=[
-          'msgpack>=0.5.4',
+          'msgpack==0.5.4',
```

Comment:
I fix the `msgpack` version here for now, they released the `0.6.0` this morning and it breaks tests.
https://github.com/msgpack/msgpack-python/pull/319

Reply:
@takluyver it seems that travis did not get the `setup.py` update for python `3.4` and `3.5` but did for `3.6`, is there some trick here you have knowledge about?

Reply:
That's weird. I've prodded it to restart the build; maybe it was just a weird temporary glitch.

The only thing I can think of is that the dependency via `msgpack-numpy` overrides the dependency we specify. It's a longstanding issue with pip that it doesn't do proper dependency resolution, so you can get strange things happening when you depend on the same package by multiple routes.

Reply:
Restarting it made it go from 1/3 passing to 0/3. :-(

The only thing I can think of is to make msgpack-numpy an optional dependency, and install it in a separate command for the tests.

Reply:
I see this is a bit annoying... Do you know if the dependencies order has in impact on this?
Thanks

Reply:
It's conceivable, but I don't know.